lib/tasks/data_maintenance.rake

```diff
@@ -28,4 +28,12 @@ namespace :data_maintenance do
   task fix_nil_allocations_estate: :environment do
     Allocation.where(estate: nil).update_all(estate: :adult_male)
   end
+
+  desc 'fix blank (empty string) person references that should be stored as null'
```

Comment:
Might this trigger a JPC/CDI catastrophe by causing all the updated_at flags of related records to be touched?

Reply:
No, Rails doesn't update the timestamps when using `.update_all` so this won't cause a hub meltdown. https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_all

> Note: As Active Record callbacks are not triggered, this method will not automatically update updated_at/updated_on columns.

Reply:
jolly good 👍